store/sqlstore/upgrade.go

```diff
@@ -303,11 +303,6 @@ func upgradeDatabaseToVersion33(sqlStore *SqlStore) {
 			}
 		}
 
-		sqlStore.CreateColumnIfNotExists("OAuthApps", "IsTrusted", "tinyint(1)", "boolean", "0")
-		sqlStore.CreateColumnIfNotExists("OAuthApps", "IconURL", "varchar(512)", "varchar(512)", "")
-
-		sqlStore.CreateColumnIfNotExists("OutgoingWebhooks", "TriggerWhen", "tinyint", "integer", "0")
```

Comment:
How is this related?

Reply:
I literally have no idea, It seems like added before as well:
https://github.com/mattermost/mattermost-server/blob/4739e93c16a0d0d884143d5f12c5ee710f6c08cb/db/migrations/mysql/000014_create_outgoing_webhooks.up.sql#L39